core/src/test/java/org/wso2/msf4j/HttpResourceModelTest.java

```diff
@@ -28,8 +28,10 @@
 
     @Test
```

Comment:
Test added to verify whether sessions are managed per service. Previous tests are applicable in the same way.